contrib/README.md

```diff
@@ -0,0 +1,11 @@
+# contrib
+
+The `contib` directory contains packages that do not belong in the core containerd packages but still contribute to the overall containerd usability.
```

Comment:
minor typo: s/contib/contrib

End of sentence minor wording suggestion: "but still contribute to overall containerd usability." (drop *the*)

Reply:
done